Products/Zuul/facades/metricfacade.py

```diff
@@ -772,9 +772,16 @@ def processRename():
                         log.info(jsonLine['content'])
                         joblog.info(jsonLine['content'])
                 elif jsonLine['type'] == 'error':
-                    log.error(jsonLine['content'])
-                    joblog.error(jsonLine['content'])
-                    nFails += 1
+                    # ZEN-29646 openTSDB API considers renaming of something what doesn't exist
+                    # as an error with status code 400, handle this as it is not an error for 
+                    # us in this particular case. 
+                    if "does not exist" not in jsonLine['content']:
+                        log.error(jsonLine['content'])
+                        joblog.error(jsonLine['content'])
+                        nFails += 1
+                    else:
+                        log.info("No metrics found for %s", oldId)
+                        joblog.info("No metrics found for %s", oldId)
```

Comment:
I don't think "oldId" is correct in this log. It is possible only certain metrics and or tagv don't have entries.  It is part of the reason that I suggested making the change to look at the response string in central query so that it could send an info or warn message instead of the error.